helm-chart/templates/_helpers.tpl

```diff
@@ -26,6 +26,10 @@
 {{- if not .Values.registry.disablePublicImages }}curlimages{{ else }}{{ .Values.registry.server }}{{ end }}
 {{- end }}
 
+{{- define "alpine.image-server" -}}
+{{- if not .Values.registry.disablePublicImages }}registry.hub.docker.com/library{{ else }}{{ .Values.registry.server }}{{ end }}
```

Comment:
Per my other comment, if we do end up needing this alpine image, we shouldn't need this full path. See the `nats` image for example, it's pulled straight from docker.io.